cerbero/packages/package.py

```diff
@@ -435,6 +432,13 @@ class MetaPackage(PackageBase):
     @type resources_distribution: string
     @cvar user_resources: folders included in the .dmg for iOS packages
     @type user_resources: list
+    @cvar osx_framework_library: name and link as a (name, relative_path)
+                                 tuple to the Framework library directory
+    @type osx_framework_library: tuple
+    @cvar osx_build_dmg: set to True (default) to move all product packages
```

Comment:
This changes the actual behaviour that creates always both the pkg and the dmg.

Reply:
With fluendo-sdk packages this behavior is dangerous because it can lead to different packages inside and outside the dmg archive, so it's a good idea to have this behavior switchable.

By default it is at True, the default/actual behavior is safe.